examples/stories/demo.story.js

```diff
@@ -0,0 +1,156 @@
+import * as skinview3d from "../../libs/skinview3d";
+import { createViewer } from "./util";
+import { radios, withKnobs, optionsKnob, number } from "@storybook/addon-knobs";
+let { viewer, element } = createViewer();
+
+let currentAnimation;
+
+const bodyParts = viewer.playerObject.skin;
+const skinParts = {};
+
+// set inner parts
+skinParts.head = bodyParts.head.innerLayer;
+skinParts.body = bodyParts.body.innerLayer;
+skinParts.leftArm = bodyParts.leftArm.innerLayer;
+skinParts.rightArm = bodyParts.rightArm.innerLayer;
+skinParts.leftLeg = bodyParts.leftLeg.innerLayer;
+skinParts.rightLeg = bodyParts.rightLeg.innerLayer;
+
+// set outter parts
+skinParts.head2 = bodyParts.head.outerLayer;
+skinParts.body2 = bodyParts.body.outerLayer;
+skinParts.leftArm2 = bodyParts.leftArm.outerLayer;
+skinParts.rightArm2 = bodyParts.rightArm.outerLayer;
+skinParts.leftLeg2 = bodyParts.leftLeg.outerLayer;
+skinParts.rightLeg2 = bodyParts.rightLeg.outerLayer;
+
+export default {
+	title: "Skinview3d",
+	decorators: [withKnobs],
+};
+
+const handleSizeControl = () => {
+	const width = number("Canvas Width", 300);
+	const height = number("Canvas Height", 400);
+
+	viewer.setSize(width, height);
+};
+
+const handleTextureControl = () => {
```

Comment:
What about using a textbox rather than radios for choosing textures? Users would be able to set a customized texture URL, or just choose a demo texture. This can be done using [`datalist`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist), but I don't know how to get it to work with storybook.